experimental/add_to_app/README.md

```diff
@@ -5,50 +5,64 @@ and the latest version of CocoaPods. See the [README](../README.md) in the
 
-* Show developers how to add Flutter to their existing applications.
+* Show developers how to add Flutter to their existing applications.install
 
 ## The important bits
 
-### `example_module`
+### Flutter modules
 
-This is the Flutter module that'll be added to all the individual projects
-in the `add_to_app` folder.
+There are two Flutter modules included in the codebase: `flutter_module` and
+`flutter_module_using_plugin`. The only significant difference between the two
+is that `flutter_module_using_plugin` depends on a Flutter plugin,
+[`path_provider`](https://pub.dev/packages/path_povider), and `flutter_module`
+ doesn't.
 
-The first time you download this repo, you need to first resolve this
-module's dependencies by running:
+Before running any of the Android or iOS apps included in this sample project,
+you first need to resolve the Flutter modules' depencies. Do so by running this
+command from within the `flutter_module` and `flutter_module_using_plugin`
+directories:
 
 ```bash
-cd example_module
 flutter packages get
 ```
 
-### `SimpleAndroidApp`
+### Android and iOS applications
 
-An Android application that imports `example_module` and uses it to display
-a `FlutterActivity` containing widgets from the module. This project brings
-in the Flutter module's Android host wrapper as a gradle sub-module.
+In addition to the Flutter modules, this repo also includes a number of
+Android and iOS applications that demonstrate different ways of importing
+them.
 
-To run, open the SimpleAndroidApp folder in Android Studio and press the run
-button.
-
-### `SimpleIOSExample`
-
-An iOS application that imports `example_module` and uses it to display
-a `FlutterViewController` containing widgets from the module. This project
-uses CocoaPods, so prior to running it for the first time, use this command
-to set up the workspace file:
+The Android apps are ready to run once you've completed the
+`flutter packages get` commands listed above. The iOS apps use CocoaPods,
+so you need to run this command to install the dependencies listed in their
+Podfiles prior to running them the first time:
 
 ```bash
-cd ../SimpleIOSExample
 pod install
 ```
 
-Then open SimpleIOSExample/SimpleIOSExample.xcworkspace with Xcode. This
-workspace will have the Flutter module linked in as a dependent CocoaPod.
+Once that command is complete, you'll find an `xcworkspace` file in the project
+directories with the correct Flutter module (and any other dependencies)
+included. Open that workspace file, and the app is ready to build and run.
+
+### `android_fullscreen` and `ios_fullscreen`
+
+These apps showcase a relatively straightforward integration of
+`flutter_module`:
+
+* The Flutter module's code is built as part of the overall app build process.
+* The engine is warmed up at app launch.
+* The Flutter view is presented with a full-screen Activity or
+  UIViewController.
+* The Flutter view can only exit; it does not launch any new, native Activities
```

Comment:
Exit may sound like you explicitly meant exiting the app. If that's not what you meant, maybe just 'pop'

Reply:
That sentence isn't all that great in general. Revised to: 

```
* The Flutter view is a navigational leaf node; it does not launch any new,
  native Activities or UIViewControllers in response to user actions.
```